### PR TITLE
Add data manager to build KMA index

### PR DESCRIPTION
**Pull request by dfornika**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by dfornika** [comment]
Error message in test logs is:

```
File "/home/travis/.planemo/planemo_tmp_58r2orh8/data_manager/kma_build_index.py", line 32, in kma_build_index
subprocess.check_call(['kma'] + ['index'] + args, cwd=target_directory)
File "/opt/python/3.7.1/lib/python3.7/subprocess.py", line 336, in check_call
retcode = call(*popenargs, **kwargs)
File "/opt/python/3.7.1/lib/python3.7/subprocess.py", line 317, in call
with Popen(*popenargs, **kwargs) as p:
File "/opt/python/3.7.1/lib/python3.7/subprocess.py", line 769, in __init__
restore_signals, start_new_session)
File "/opt/python/3.7.1/lib/python3.7/subprocess.py", line 1516, in _execute_child
raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'kma': 'kma'
```

**Comment by dfornika** [comment]
I also noticed this line in the test output:

```
WARNI [galaxy.tool_util.deps.installable] Conda not installed and auto-installation disabled.
```

maybe that explains why `kma` isn't being found?

File: data_managers/data_manager_build_kma_index/data_manager/kma_build_index.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<tool id="kma_build_index" name="KMA Index Builder" tool_type="manage_data" version="1.2.21+galaxy0">
```
Comment (mvdbeek) [inline data_managers/data_manager_build_kma_index/data_manager/kma_build_index.xml]:
```suggestion
<tool id="kma_build_index" name="KMA Index Builder" tool_type="manage_data" version="1.2.21+galaxy0" profile="19.01">
```
that will fix the installation. Please always include a recent profile version in all new tools.

**Comment by bgruening** [comment]
@dfornika merci! :)))

**Pull request merged**